workflows/wf3/1-fetch-metadata.js

```diff
@@ -38,10 +24,31 @@ fn(state => {
   return state;
 });
 
+get(
+  `${$.workbookBase}/worksheets('omrs-form-metadata')/usedRange`,
+  {},
+  state => {
+    const [headers, ...rows] = state.data.values.slice(0);
+    state.formMetadata = rows
+      .map(row =>
+        row.reduce((obj, value, index) => {
+          if (value != null && value !== '') {
```

Comment:
You should be able to just do `if(value)` here, no?

Reply:
I will have a look